core/Services/Calendar/Store.vala

```diff
@@ -425,7 +424,8 @@ public class Calendar.Store : Object {
 #endif
         debug (@"Received $(objects.length()) added event(s) for source '%s'", source.dup_display_name ());
         var events = source_events.get (source);
-        var added_events = new Gee.ArrayList<ECal.Component> ((Gee.EqualDataFunc<ECal.Component>?) Calendar.Util.ecalcomponent_equal_func);
+        var added_events = new Gee.ArrayList<ECal.Component> ((Gee.EqualDataFunc<ECal.Component>?) Util.ecalcomponent_equal_func);
```

Comment:
We should always use the full namespace path (`Calendar.Util.ecalcomponent_equal_func`) to avoid confusions with the (for now) co-existing `Maya.Util`.

Reply:
Done